commands/src/main/java/org/wildfly/extras/creaper/commands/transactions/ChangeJdbcTransactionAttributes.java

```diff
@@ -0,0 +1,170 @@
+package org.wildfly.extras.creaper.commands.transactions;
+
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.ServerVersion;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+public final class ChangeJdbcTransactionAttributes implements OfflineCommand, OnlineCommand {
+
+    private final Boolean useJdbcStore;
+    private final Boolean actionStoreDropTable;
+    private final String actionStoreTablePrefix;
+    private final Boolean communicationStoreDropTable;
+    private final String communicationStoreTablePrefix;
+    private final String stateStoreTablePrefix;
+    private final String storeDatasource;
+    private final Boolean stateStoreDropTable;
+
+    protected ChangeJdbcTransactionAttributes(Builder builder) {
+        useJdbcStore = builder.useJdbcStore;
+        actionStoreDropTable = builder.actionStoreDropTable;
+        actionStoreTablePrefix = builder.actionStoreTablePrefix;
+        communicationStoreDropTable = builder.communicationStoreDropTable;
+        communicationStoreTablePrefix = builder.communicationStoreTablePrefix;
+        stateStoreTablePrefix = builder.stateStoreTablePrefix;
+        storeDatasource = builder.storeDatasource;
+        stateStoreDropTable = builder.stateStoreDropTable;
+    }
+
+    @Override
+    public void apply(OfflineCommandContext ctx) throws Exception {
+        if (ctx.version.lessThanOrEqualTo(ServerVersion.VERSION_3_0_0)) {
```

Comment:
By the way, if you have offline commands that are also version-specific, it would be a good idea to add a check like this too, now that offline version discovery works.